functions/helloworld/hello_logging.go

```diff
@@ -31,8 +31,8 @@ var (
 
 // HelloLogging logs messages.
 func HelloLogging(w http.ResponseWriter, r *http.Request) {
-	stdLogger.Println("I am a log entry!")
-	logger.Println("I am an error!")
+	stdLogger.Println("This is sent to os.Stdout")
+	logger.Println("This is sent to os.Stderr")
```

Comment:
maybe outLogger and errLogger?

Reply:
or ...

log.Println("This is stderr")
fmt.Println("This is stdout")

log format with timestamps doesn't really matter for logging because the logging service attaches its own timestamp

Reply:
Done.